### PR TITLE
GraphAPIError getting the error_subcode from the request and put logging for debugging purpose

### PR DESCRIPTION
**Pull request by kayran**
The Facebook are returning a important information in some requests, getting the error subcode. This Pull request add the treatment to get this info and return in GraphAPIError

### CODE REVIEW

File: facebook/__init__.py [hunk 1]
```diff
@@ -255,6 +255,12 @@ def request(
                 args["access_token"] = self.access_token
 
         try:
+            import logging
```
Comment (martey) [inline facebook/__init__.py]: https://www.python.org/dev/peps/pep-0008/#imports

File: facebook/__init__.py [hunk 2]
```diff
@@ -255,6 +255,12 @@ def request(
...
+            logging.info('GRAPH REQUEST URL:%s'% (FACEBOOK_GRAPH_URL + path))
+            logging.info('GRAPH REQUEST params:')
+            logging.info(args)
+            logging.info('GRAPH REQUEST data:')
+            logging.info(post_args)
```
Comment (martey) [inline facebook/__init__.py]: Please read the [contributing guidelines](https://github.com/mobolic/facebook-sdk/blob/master/CONTRIBUTING.rst).

> A pull request should only include commits directly applicable to its change.

Comment (kayran) [reply facebook/__init__.py]: thanks @martey

**Comment by kayran** [comment]
@martey You have anyone change to suggest?

File: facebook/__init__.py [hunk 3]
```diff
@@ -255,6 +256,12 @@ def request(
                 args["access_token"] = self.access_token
 
         try:
+
+            logging.info('GRAPH REQUEST URL:%s' % (FACEBOOK_GRAPH_URL + path))
+            logging.info('GRAPH REQUEST params:')
+            logging.info(args)
+            logging.info('GRAPH REQUEST data:')
+            logging.info(post_args)
```
Comment (martey) [inline facebook/__init__.py]: This should be removed from here and added in another pull request. Adding logging has nothing to do with error handling.

**Comment by DmytroLitvinov** [comment]
Hi @martey,

Seems like this PR is ready to be merged.

**Comment by DmytroLitvinov** [comment]
[This PR](https://github.com/mobolic/facebook-sdk/pull/453) with changes in CHANGELOG.

**Comment by kayran** [comment]
@martey all logs are removed, and the checks are working, can you accept this?

**Pull request closed without merging**